packages/apollo-server/README.md

```diff
@@ -122,17 +122,17 @@ new ApolloServer({
 
 ## Documentation
 
-The [Apollo Server documentation](https://apollographql.com/docs/apollo-server/) contains many of the details which are necessary to get started with both GraphQL and Apollo Server.
+The [Apollo Server documentation](https://apollographql.com/docs/apollo-server/) contains much more to get started with GraphQL and Apollo Server.
```

Comment:
Rather than _contains much more_, how about?:

```suggestion
The [Apollo Server documentation](https://apollographql.com/docs/apollo-server/) contains additional details on how to get started with GraphQL and Apollo Server.
```